cmd/fanlin/main.go

```diff
@@ -101,12 +102,18 @@ func main() {
 	}
 
 	http.DefaultClient.Timeout = conf.BackendRequestTimeout()
-
 	runtime.GOMAXPROCS(conf.MaxProcess())
+	mux := http.DefaultServeMux
```

Comment:
Can you write like this without using DefaultServeMux.

```
	var h http.Handler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
		handler.MainHandler(w, r, conf, loggers)
	})

	if conf.UseServerTiming() {
		h = servertiming.Middleware(h, nil)
	}

	http.Handle("/", h)
	http.HandleFunc("/healthCheck", handler.HealthCheckHandler)

	http.ListenAndServe(fmt.Sprintf(":%d", conf.Port()), nil)
```

Reply:
OK, I simplify it like before.